backend/LexBoxApi/Services/PermissionService.cs

```diff
@@ -94,4 +94,17 @@ public void AssertHasProjectRequestPermission()
     {
         if (!HasProjectRequestPermission()) throw new UnauthorizedAccessException();
     }
+
+    public void AssertCanCreateOrg()
+    {
+        if (!HasProjectCreatePermission()) throw new UnauthorizedAccessException();
```

Comment:
This obviously isn't set in stone, but...I'd expect we want this to be a lot stricter at the beginning.

I expect that the main orgs we need will be represented by current admins and to prevent duplicate orgs from being created we'll need to have process/verification in place. So, I'm thinking this will be admin only, at least initially.

Reply:
Yeah that makes sense too, I'm not sure the best way to avoid duplicates right now. Probably we search for orgs while they type in the org name to create one and suggest that they don't create a new one.